test/test.js

```diff
@@ -323,6 +323,63 @@ describe('csurf', function () {
     })
   })
 
+  describe.only('req.verifyToken()', function () {
+    var app, token, cookie;
```

Comment:
Rather than closing over variables, please take advantage of mocha and attach to "this"